Create Sondage Rattrapages - Semestre impair 2022-2023.md

gtedavid opened the pull request:
⚠
Éléments à changer :

- [x] format utilisé dans le message;
- [x]  canaux mentionnés 


A vérifier :

- [x]  les rôles utilisés par <@&>

Le formatage md utilisé provient d'une version précédente du bot, plus utilisé. [Se référer au README](https://github.com/horizon-efrei/efreussite-webhooks/#readme)

Les canaux mentionnés sont du Semestre pair et non impair. Le post provient du sondage réalisé l'été dernier.

gtedavid committed:
Create Sondage Rattrapages - Semestre impair 2022-2023.md

Elements to change :
- Format ;
- channel names

The format used is one of a previous bot version.


Éléments à changer :
- format utilisé dans le message;
- canaux mentionnés

A vérifier :
- les rôles utilisés par <@&>

Le formatage md utilisé provient d'une version précédente du bot, plus utilisé.
[Se référer à](https://github.com/horizon-efrei/efreussite-webhooks/#readme)

gtedavid commented:
A ne pas merge avant que les changements sur le fichier ne soient réalisés

gtedavid committed:
Update Sondage Rattrapages - Semestre impair 2022-2023.md

Éléments remplacés :
` ` -> ** **
2 lignes pour séparer les msg -> ===MESSAGE_BREAK===

gtedavid committed:
Remove L3 dates 

RAT L3 sont en juin

SaadaLaura commented on resources/poll/2022-2023/Sondage Rattrapages - Semestre impair 2022-2023.md:
```diff
@@ -0,0 +1,47 @@
+** **
+__**RATTRAPAGES L1**__
+
+:zero: <#796439254469312582>
+:one: <#927862574777790495>
+:two: <#977913803385749504>
+:three: <#694636294961234001>
+:four: <#694636933166530580>
+:five: <#809466892065832980>
+:six: <#809084853348991017>
+:seven: <#819540421545033759>
+:eight: <#802867138518843432>
+:nine: <#927857179824889896>
```
:zero: <#761922885244813312>
:one: <#819658918648479786>
:two: <#796439367878967297>
:three: <#796439254469312582>
:four: <#885636868362895411>
:five: <#799611218352406528>
:six: <#809466892065832980>

Si vous avez des questions pour le rattrapage de comm ou d'anglais vous pouvez les poser sur les chanels respectifs (<#694636294961234001> et <#694636933166530580>)

gtedavid replied:
Si vous avez des questions pour les rattrapages de communication ou d'anglais, vous pouvez les poser sur les canaux respectifs (<#694636294961234001> et <#694636933166530580>)

gtedavid replied:
Les matières sont organisées par la date du premier rattrapage ?

SaadaLaura replied:
Oui les réactions sont dans l'ordre des dates pourquoi il y a un pb avec ça ?

gtedavid replied:
Nope
ça devrait le faire...

gtedavid replied:
tu peux faire les modifications sur le fichier pour le coup

SaadaLaura replied:
Oki oki, pour le message pour la comm et l'anglais soit on le met apres les dates de L2 soit on le met en dessous des L1 et en dessous des L2

gtedavid replied:
> on le met après les dates de L2

c'est le mieux je pense.....
Après comme tu le sens

gtedavid replied:
ah

SaadaLaura replied:
> tu peux faire les modifications sur le fichier pour le coup

comment ça ? :sweat_smile:

SaadaLaura replied:
> > on le met en dessous des L1 et en dessous des L2
> 
> c'est le mieux je pense..... Après comme tu le sens

Oki no pb les deux me vont

gtedavid replied:
> > > on le met en dessous des L1 et en dessous des L2
> > 
> > 
> > c'est le mieux je pense..... Après comme tu le sens
> 
> Oki no pb les deux me vont

J'ai modifié la citation, je m'étais trompé 😅

SaadaLaura replied:
> J'ai modifié la citation, je m'étais trompé 😅

tkt tkt 😅 j'avoue que on était en décalé dans les message

gtedavid replied:
> > tu peux faire les modifications sur le fichier pour le coup
> 
> comment ça ? 😅

Là ce que tu fais, c'est des propositions de review suivi de commentaires

SaadaLaura replied:
> > > tu peux faire les modifications sur le fichier pour le coup
> > 
> > 
> > comment ça ? 😅
> 
> Là ce que tu fais, c'est des propositions de review suivi de commentaires

Oui car je n'étais pas sur que tu sois ok pour la comm

SaadaLaura commented on resources/poll/2022-2023/Sondage Rattrapages - Semestre impair 2022-2023.md:
```diff
@@ -0,0 +1,47 @@
+** **
+__**RATTRAPAGES L2**__
+
+:zero: <#815938394013827112>
+:one: <#809082397781655592>
+:two: <#819374104108728340>
+:three: <#694636294961234001>
+:four: <#888085892080738324>
+:five: <#927858681972944926>
+:six: <#832302872506138684>
+:seven: <#694636933166530580>
+:eight: <#809082131166134313>
+:nine: <#761922623063064648>
+
+===MESSAGE_BREAK===
+** **
+:zero: <#888085892080738324>
+:one: <#809818898182045696>
+:two: <#810151770192871454>
```
:zero: <#885809358108262420>
:one: <#809082397781655592>
:two: <#761923002459488277>
:three: <#761923165642948619>
:four: <#885613800747831396>
:five: <#885610098037252196>
:six: <#816658561618411542>
:seven: <#818891095815618590>
:eight: <#761923093231960075>
:nine: <#779715353814630410>

===MESSAGE_BREAK===
** **
:zero: <#797220848033464410>

Si vous avez des questions pour le rattrapage de comm ou d'anglais vous pouvez les poser sur les chanels respectifs (<#694636294961234001> et <#694636933166530580>)

SaadaLaura replied:
Par contre je ne sais pas si on en fait une pour "révolution numérique et entreprenariat" je la met au cas ou mais on peut la supprimer

gtedavid committed:
Update roles & infos

- MAJ rôles
- MAJ msg
- MSG info rôles menus dans le message

gtedavid committed:
Format correction + lines

- Suppression de l'envoi d'un msg d'une ligne (fusion de deux messages)
- Rajout de sauts de lignes
